src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -379,13 +379,21 @@ export class ExtensionEditor extends BaseEditor {
 			template.license.style.display = 'none';
 		}
 
-		// {{SQL CARBON EDIT}} add license url
-		if (extension.licenseUrl) {
-			template.license.onclick = finalHandler(() => window.open(extension.licenseUrl));
-			template.license.style.display = 'initial';
-		} else {
-			template.license.onclick = null;
-			template.license.style.display = 'none';
+		// {{SQL CARBON EDIT}}
+		// copied from the the extension.url condition block above
+		// for ADS the extension.url will be empty but we still want to make the publisher and license controls to be clickable
+		if (!extension.url) {
+			if (extension.licenseUrl) {
+				this.transientDisposables.add(this.onClick(template.license, () => this.openerService.open(URI.parse(extension.licenseUrl!))));
+				template.license.style.display = 'initial';
+			} else {
+				template.license.style.display = 'none';
+			}
+			this.transientDisposables.add(this.onClick(template.publisher, () => {
+				this.viewletService.openViewlet(VIEWLET_ID, true)
+					.then(viewlet => viewlet as IExtensionsViewlet)
+					.then(viewlet => viewlet.search(`publisher:"${extension.publisherDisplayName}"`));
```

Comment:
You can just defined the viewlte in the param, `(viewlet: IExtensionsViewlet) => `